CONTRIBUTING.md

```diff
@@ -0,0 +1,105 @@
+# Contributing to Arkime AWS All In One
+
+:sparkles: Glad to see you here! :sparkles:
+
+---
+
+### Just have a question :question:
+
+* Visit our [FAQs](https://arkime.com/faq)
+* Or talk to us directly in the [Arkime Slack](https://slackinvite.arkime.com/)
+
+---
+
+### Where do I start? :traffic_light:
+
+First, checkout the main [Arkime AWS AIO README](README.md) for information on how to build and run Arkime. We do all development on MacBook Pros.
+
+#### How do I contribute?
+
+#### Documentation! :page_with_curl:
+
+Documentation, READMEs, examples, and FAQs are important. Please help improve and add to them.
+
+#### Bugs :bug: :beetle: :ant:
+
+**Before submitting a bug report:**
+* Ensure the bug was not already reported by searching for [existing issues in Arkime AWS AIO](https://github.com/arkime/aws-aio/issues)
+  * If an issues is already open, make a comment that you are experiencing the same thing and provide any additional details
+* Check the [FAQs](https://arkime.com/faq) for a list of common questions and problems
+
+Bugs are tracked as [GitHub Issues](https://guides.github.com/features/issues/).
+**Please follow these guidelines when submitting a bug:**
+* Provide a clear and descriptive title
+* Describe the exact steps to reproduce the problem
+* Explain the expected behavior
+
+#### Feature Requests :sparkles:
+
+Feature requests include new features and minor improvements to existing functionality.
+
+Feature requests are tracked as [GitHub Issues](https://guides.github.com/features/issues/).
+**Please follow these guidelines when submitting a feature request:**
+* Please use a [fork](https://guides.github.com/activities/forking/) to submit a [pull request](https://help.github.com/articles/creating-a-pull-request/) for your contribution.
+* Provide a clear and descriptive title
+* Describe the suggested feature in as much detail as possible
+* Use examples to help us understand the use case of the feature
+* If you are requesting a minor improvement, describe the current behavior and why it is not sufficient
+* If possible, provide examples of where this feature exists elsewhere in other tools
+
+#### Pull Requests :muscle:
+
+**We welcome all collaboration!** If you can fix it or implement it, please do! :hammer:
+To implement something new, please create an issue first so we can discuss it together.
+
+**To better help us review your pull request, please follow these guidelines:**
+* Provide a clear and descriptive title
+* Clearly describe the problem and solution
+* Include the relevant issue number(s) if applicable
+* Run unit tests and lint as [described below](how-to-run-the-unit-tests-&-lint)
```

Comment:
I think we have to have a higher bar, here.  There are a lot of subtle ways things could break that are outside the purview of the unit tests.  For example - there could be a mismatch between the Python and CDK sides on a passed context variable.  In all my own PRs, I post the CLI output from the commands I run to test the changes actually work, and I'd feel more comfortable if that's the bar we maintained for all PRs.

How about:
```
We require the unit tests and Python/TypeScript linters to pass before merging PRs.  Additionally, any substantive changes should include:
* A clear explanation of what automated and manual testing you performed
* The output from the CLI of the test commands you ran to confirm the change operated as expected
* Screenshots of the AWS Console/Arkime Dashboard/etc as appropriate to demonstrate the change operates as expected
```

In other words - just having passing unit tests and linters doesn't tell us much unless the change is completely trivial.

Thoughts?